table_name.go

```diff
@@ -66,12 +66,16 @@ func (f *font) GetNameByID(nameID int) string {
 }
 
 // GetNameRecords returns name records as slice map.
```

Comment:
would be good to add what is the structure of the map, not very clear
seems like language id : nameId : value ?

Reply:
updated at 5f17edba287318797a191da51f9ff90b9b23c82c and added test case.